README.md

```diff
@@ -73,3 +73,41 @@ client.on('incoming', (transfer) => {
   client.fulfillCondition(transfer.id, 'cf:0:')
 })
 ```
+
+## Extensions
+
+To extend the functionality of the `Client`:
+
+```js
+import { Client } from 'ilp-core'
+
+class MyExtension {
+  constructor (client) {
+    this.client = client
+    this.client.on('receive', this._handleReceive.bind(this))
+  }
+
+  static getName () {
+    return 'extension'
+  }
+
+  doSomething () {
+    console.log('hello there')
```

Comment:
Maybe show an example of interacting with the client. I like that you have an example of listening to an event, this would close the loop for me logically.